docs/src/guide/installation.md

```diff
@@ -250,6 +251,79 @@ Download and run the [Installer][doxypress_binaries].
+
+- Install Ubuntu-22.04 LTS as WSL2 distribution
+  - See the [general WSL doc](https://docs.microsoft.com/en-us/windows/wsl/install), but
+  - as of this writing `wsl --list --online` will not list Ubuntu 22.04.1 LTS, but it is [available in the Microsoft Store](https://www.microsoft.com/store/productId/9PN20MSR04DW). Install it from there via mouse clicks
+- Install [Visual Studio Code](https://code.visualstudio.com/) and the [remote extensions for WSL](https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-vscode) and make sure you can log in to your linux distribution
+- Physically attach the microcontroller board to a USB port
+- Make sure the WSL2 instance is running, either by opening a terminal via start menu or by using the [VSCode integrated terminal](https://code.visualstudio.com/docs/remote/wsl-tutorial#_integrated-terminal). 
+- [Bridge the USB device to Linux](https://docs.microsoft.com/en-us/windows/wsl/connect-usb) using [usbipd](https://github.com/dorssel/usbipd-win)
+  - This works for Windows 11 and Windows 10. Your sesssion might look similar to this one:
+    First find the USB port to be bridged
+    ```powershell
+	PS C:\Windows\system32> usbipd wsl list
+    BUSID  VID:PID    DEVICE                                                        STATE
+    3-2    1bcf:0005  USB-Eingabegerät                                              Not attached
+    3-3    0483:374b  ST-Link Debug, USB-Massenspeichergerät, Serielles USB-Ger...  Attached - Ubuntu-22.04
+    4-1    138a:003d  Synaptics FP Sensors (WBF) (PID=003d)                         Not attached
+    4-3    04f2:b370  HP HD Webcam [Fixed]                                          Not attached
+    ```
+	Check the BUSID parameter and use the one that corresponds to your device in further commands. In this example it is the `3-3`.
+	```powershell
+    PS C:\Windows\system32> usbipd wsl attach --busid 3-3
+    usbipd: info: Using default distribution 'Ubuntu-22.04'.
+    ```
+    Note: the system is not very reliable, it might be necessary to detach/attach several times until the connection is established
+    ```powershell
+	PS C:\Windows\system32> usbipd wsl detach --busid 3-3
+	PS C:\Windows\system32> usbipd wsl attach --busid 3-3
+    usbipd: info: Using default distribution 'Ubuntu-22.04'.
+	``` 
+  - In WSL2 Ubuntu-22.04 configure the USB ports as user accessible. This requires the configuration of `udev` rules. 
+    - Use an editor to create a new `udev` rules file with a high enough number in the filenam, e.g. `sudo nano /etc/udev/rules.d/100-udev-stm.rules`
+    - Add some rules similar to [the ones used in OpenOCD](https://github.com/openocd-org/openocd/blob/master/contrib/60-openocd.rules#L84-L99), e.g.
+      ```
+	  # STMicroelectronics ST-LINK V1
```

Comment:
1. There should be no need to create any additional custom udev rules since [Ubuntus OpenOCD package contains](https://packages.ubuntu.com/jammy/amd64/openocd/filelist) the udev rule file `/lib/udev/rules.d/60-openocd.rules`. 
2. modm supports not just STM32, so (if necessary, see above) just copy the whole file.

Reply:
> 1. There should be no need to create any additional custom udev rules since [Ubuntus OpenOCD package contains](https://packages.ubuntu.com/jammy/amd64/openocd/filelist) the udev rule file `/lib/udev/rules.d/60-openocd.rules`.

Weird enough it only started to work after adding it under `/etc/udev/`. Maybe @joelsa has an explanation?

Reply:
> 2\. modm supports not just STM32, so (if necessary, see above) just copy the whole file.

The file is rather long. Do you want me to copy the whole content into the text?